src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -234,6 +234,15 @@ private static IEnumerable<ISetupInstance> EnumerateVisualStudioInstances()
         private static ISetupInstance LocateVisualStudioInstance(ImmutableHashSet<string> requiredPackageIds)
         {
             var vsInstallDir = Environment.GetEnvironmentVariable("VSInstallDir");
+            if (vsInstallDir is null)
+            {
+                vsInstallDir = Environment.GetEnvironmentVariable("__UNITTESTEXPLORER_VSINSTALLPATH__");
```

Comment:
Why not prefer this first? If I launch VS from a developer command prompt (which I do regularly), VSInstallDir will be set and this codepath never considered.